47.md

```diff
@@ -83,9 +121,9 @@ If the command was successful, the `error` field must be null.
 
 ### Notification Events
 
-The notification event SHOULD contain one `p` tag, the public key of the **client**.
+The notification event is a kind 23197 event (23196 if using nip04) SHOULD contain one `p` tag, the public key of the **user**.
```

Comment:
```suggestion
The notification event is a kind 23197 event (23196 if using nip04) SHOULD contain one `p` tag, the public key of the **client**.
```

Reply:
Also I'm wondering if we even mention 23196 here since we have the note on backward compatibility below?

Reply:
Done and removed that reference too.